justfile

```diff
@@ -317,6 +318,7 @@ linkerd-install *args='': linkerd-load linkerd-crds-install && _linkerd-ready
             --set='proxy.image.version={{ linkerd-tag }}' \
             --set='proxyInit.image.name={{ proxy-init-image }}' \
             --set="proxyInit.image.version=$(yq .proxyInit.image.version charts/linkerd-control-plane/values.yaml)" \
+            --set="cniPluginVersion=$(yq .cniPluginVersion charts/linkerd-cni/values.yaml)" \
```

Comment:
`cniPluginVersion` is a var in the `linkerd2-cni` helm chart, so it's used when installing that chart directly or through the subcommand `linkerd install-cni`, but not when simply doing `linkerd install`.